tests/test_plc_route.py

```diff
@@ -53,13 +53,13 @@ def plc_route_reciever(self):
 
             data = data[4:]  # Remove protocol bytes
 
-            len_sending_host = struct.unpack("<H", data[:2])[0]  # Length of host name
+            len_route_name = struct.unpack("<H", data[:2])[0]  # Length of PLC password
```

Comment:
route name or plc password? There is a contradiction here between the comment and the variable name